src/business-logic/services/interfaces/inewsgateway-device-connection.ts

```diff
@@ -0,0 +1,15 @@
+import { INewsGatewayDeviceConnection } from '../inews-gateway-connection-implementation'
+
+export interface DeviceConnection {
+  connect(): Promise<boolean>
+  disconnect(): Promise<boolean>
+  send(deviceId: string, params: string[]): Promise<void>
```

Comment:
I started getting mental fatigue at the end of last week and decided to ignore what kind of paramList I needed and just use string array instead, then refactor later when I wasn't as tired. This will be addressed in an upcoming push.